content/troubleshooting/java/installation/NoClassDefFoundError.md

```diff
@@ -9,9 +9,8 @@ tags: "troubleshoot java agent NoClassDefFoundError"
 ```java.lang.NoClassDefFoundError:com/.../EventController
 ```
 
-This error indicates you've renamed your agent jar from its original name, ***contrast.jar***. 
-
+This error usually indicates you've renamed your agent jar from its original name, ***contrast.jar***. It may also mean that you're using an OSGi container like Felix or Equinox that is having trouble working with our auto-updating agent.
```

Comment:
Did we test equinox?

Reply:
Yes, equinox is tested as part of WSO2 / Synapse.

Reply:
Do you know why that test didn't catch this bug?